setup.cfg

```
@@ -23,7 +23,7 @@ description = A workflow engine for cycling systems
 long_description=file: README.md
 long_description_content_type=text/markdown
 project_urls =
-    Documentation = https://cylc.org/cylc-doc
+    Documentation = https://cylc.github.io/cylc-doc
```

surely https://cylc.github.io/cylc-doc/latest/html/index.html

---

This URL is shown on the PyPi page so we would want it to show "stable" rather than "latest", however, since "stable" is currently Cylc 7 and we don't have any Cylc 7 releases on PyPi, fair enough.

```suggestion
    Documentation = https://cylc.github.io/cylc-doc/latest/html/index.html
```